src/app/dashboard/news/[id]/NewsDetailClient.js

```
@@ -0,0 +1,391 @@
+                                editorState={contentState}
+                                onEditorStateChange={setContentState}
+                                toolbar={editorToolbar}
+                                editorClassName="px-3 min-h-[400px]"
+                                wrapperClassName="editor-wrapper"
+                                stripPastedStyles={true}
+                            />
+                        </div>
+                    ) : (
+                        <div 
+                            className="border border-gray-300 p-4 rounded-md"
+                            dangerouslySetInnerHTML={{ __html: newsData.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)